kyo-core/shared/src/main/scala/kyo/chunks.scala

```diff
@@ -81,7 +81,7 @@ sealed abstract class Chunk[T] derives CanEqual:
         end if
     end concat
 
-    final def map[U: Flat, S](f: T => U < S): Chunk[U] < S =
```

Comment:
`Flat` wasn't necessary here. The `Loops` call needs a proof that `Chunk[U]` is flat, which is a given, not that `U` is flat.